workflow/tests/hpxml_translator_test.rb

```diff
@@ -1019,6 +1019,11 @@ def _verify_outputs(rundir, hpxml_path, results, hpxml)
       if subsurface.is_a? HPXML::Skylight
         hpxml_value /= 1.2 # Convert from NFRC 20-degree slope to vertical position
       end
+      if (not subsurface.storm_window_type.nil?) && (subsurface.is_a? HPXML::Window)
+        hpxml_value = 0.294 # U-Factor adjusted by Constructions.get_ufactor_shgc_adjusted_by_storms
+      elsif (not subsurface.storm_window_type.nil?) && (subsurface.is_a? HPXML::Skylight)
+        hpxml_value = 0.3672 / 1.2 # U-Factor adjusted by Constructions.get_ufactor_shgc_adjusted_by_storms; converted to the 20-deg slope from the vertical position by multiplying the tested value at vertical
+      end
```

Comment:
I suggest just calling `Constructions.get_ufactor_shgc_adjusted_by_storms` directly here (and below for skylights). This isn't really the right place to be hard-coding expected values, we're just trying to ensure here that the EnergyPlus outputs match the HPXML inputs.

To test expected values, I suggest adding tests to `test_enclosure.rb` where you check that you get the adjusted value you expect when a storm is present. Those tests can be A) direct calls to `Constructions.get_ufactor_shgc_adjusted_by_storms` and asserting that the resulting values are the values you listed above, or B) looking at the resulting OpenStudio model and asserting that the window (subsurface) objects have the expected ufactor/shgc.